src/LaravelChatApiServiceProvider.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Saritasa\LaravelChatApi;
+
+use Illuminate\Support\ServiceProvider;
+use Saritasa\LaravelChatApi\Contracts\IChatService;
+use Saritasa\LaravelChatApi\Contracts\INotificationsFactory;
+use Saritasa\LaravelChatApi\Notifications\NotificationsFactory;
+use Saritasa\LaravelChatApi\Services\ChatService;
+
+class LaravelChatApiServiceProvider extends ServiceProvider
+{
+    /**
+     * Bootstrap the application services.
+     *
+     * @return void
+     */
+    public function boot(): void
```

Comment:
You can also register API routes here instead of application, consuming this library